curriculum/challenges/english/08-coding-interview-prep/rosetta-code/9-billion-names-of-god-the-integer.english.md

```diff
@@ -6,15 +6,17 @@ challengeType: 5
 
 ## Description
 <section id='description'>
-<p>This task is a variation of the <a href="https://en.wikipedia.org/wiki/The Nine Billion Names of God#Plot_summary" title="wp: The Nine Billion Names of God#Plot_summary">short story by Arthur C. Clarke</a>.</p>
-<p>(Solvers should be aware of the consequences of completing this task.)</p>
-<p>In detail, to specify what is meant by a  “name”:</p>
-<p>The integer 1 has 1 name “1”.</p>
-<p>The integer 2 has 2 names “1+1”, and “2”.</p>
-<p>The integer 3 has 3 names “1+1+1”, “2+1”,  and “3”.</p>
-<p>The integer 4 has 5 names “1+1+1+1”, “2+1+1”, “2+2”, “3+1”, “4”.</p>
-<p>The integer 5 has 7 names “1+1+1+1+1”, “2+1+1+1”, “2+2+1”, “3+1+1”, “3+2”, “4+1”, “5”.</p>
-<p>This can be visualized in the following form:</p>
+This task is a variation of the <a href='https://en.wikipedia.org/wiki/The Nine Billion Names of God#Plot_summary' title='wp: The Nine Billion Names of God#Plot_summary' target='_blank'>short story by Arthur C. Clarke</a>.
+(Solvers should be aware of the consequences of completing this task.)
+In detail, to specify what is meant by a  “name”:
+<ul>
+  <li>The integer 1 has 1 name “1”.</li>
```

Comment:
@scissorsneedfoodtoo Should we keep the indentation? Does the `<pre>` section problem apply here?

Reply:
@NitronR, I think we should keep it to help with readability. The `<pre>` section problems don't apply here.